src/main/java/Gateway/DatabaseConnect.java

```diff
@@ -208,7 +208,7 @@ public Hashtable<Integer, List<Object>> loadAllReviewsGot(String usrname) throws
         String currname;
         for (int i = 1; i < sheet.getPhysicalNumberOfRows(); i++) {
             currname = sheet.getRow(i).getCell(0).toString();
-            if ((currname.equals(usrname)) & (sheet.getRow(i).getPhysicalNumberOfCells() == 4)) {
+            if ((currname.equals(usrname)) & (sheet.getRow(i).getPhysicalNumberOfCells() == 4) & (sheet.getRow(i).getCell(3).getNumericCellValue() != 0.0)) {
```

Comment:
add an additional check condition because sometimes the last cell is empty but `getPhysicalNumberOfCells` still returns 4. This is just a double check in case `getPhysicalNumberOfCells` fails